schema-generic.graphql

```diff
@@ -6,16 +6,21 @@ enum Network {
   ARBITRUM_ONE
   AURORA
   AVALANCHE
+  BOBA
   BSC # aka BNB Chain
   CELO
+  CHAPEL
+  CLOVER
   MAINNET # Ethereum Mainnet
   FANTOM
   FUSE
   MOONBEAM
   MOONRIVER
+  MUMBAI
```

Comment:
isn't mumbai a testnet?

Reply:
yes you are right, polygon testnet. removed